### DISCUSSION TITLE
colobot use all my processor

### DISCUSSION THREAD

**Discussion by pedronatorBzh** [colobot/colobot#1486] [Q&A]
the game use 100% of my processor on debian in 1.2 and 2.0

**Comment by tomangelo2** [comment]
Can you provide more details on your hardware (CPU/GPU model)? Do you have VSync on? How much FPS do you have (F12 key)?

**Comment by pedronatorBzh** [comment]
I have a ryzen 5 4500 U with radeon graphics
in the main menu and in game, I have 60fps
in Htop colobot use 4x more CPU than google chrome
